src/Gregwar/Captcha/CaptchaBuilder.php

```diff
@@ -348,6 +348,11 @@ protected function writePhrase($image, $phrase, $font, $width, $height)
         $x = ($width - $textWidth) / 2;
         $y = ($height - $textHeight) / 2 + $size;
 
+        // Define a empty array when textColor is null
+        if(is_null($this->textColor)){
+            $this->textColor = array();
+        }
+
         if (!$this->textColor) {
```

Comment:
```suggestion
        if ($this->textColor === null) {
```

Reply:
So when textColor always be an array, why should it compared with null?
The if is for empty arrays i think.

Reply:
I see you edit it. `!$this->textColor` should be enough. If `$this->textColor` is null its also true.